apps/web/src/pages/Applications/ApplicationSelfDeclarationPage/ApplicationSelfDeclarationPage.tsx

```diff
@@ -0,0 +1,427 @@
+                    mode="solid"
+                    value="explore"
+                    {...actionProps}
+                    onClick={() => {
+                      setValue("action", "explore");
+                    }}
+                  >
+                    {intl.formatMessage(
+                      {
+                        defaultMessage:
+                          "Explore <abbreviation>IT</abbreviation> opportunities within the federal government",
+                        id: "j3WBqJ",
+                        description:
+                          "Button text to submit the Indigenous self-declaration form when not Indigenous.",
+                      },
+                      {
+                        abbreviation: (text: React.ReactNode) =>
+                          wrapAbbr(text, intl),
+                      },
+                    )}
+                  </Button>
+                </p>
+              </>
+            )}
+          </div>
+          <Separator
+            orientation="horizontal"
+            decorative
+            data-h2-background-color="base(secondary)"
+            data-h2-margin="base(x2, 0)"
+          />
+          <HelpLink />
+          <p data-h2-font-weight="base(700)" data-h2-margin="base(x1, 0)">
+            {intl.formatMessage(
+              {
+                defaultMessage:
+                  "See <whyLink>why we are asking you to self declare</whyLink>, <verificationLink>how this will be verified</verificationLink> and the term <definitionLink>Indigenous as defined for this program</definitionLink>.",
+                id: "AMboRG",
+                description:
+                  "Links to more information on the self-declaration process and definition of Indigenous",
+              },
+              {
+                whyLink,
+                verificationLink,
+                definitionLink,
+              },
+            )}
+          </p>
+        </form>
+      </FormProvider>
+    </>
+  );
+};
+
+const ApplicationSelfDeclarationPage = () => {
+  const intl = useIntl();
+  const paths = useRoutes();
+  const { applicationId } = useParams();
+  const [
+    {
+      data: applicationData,
+      fetching: applicationFetching,
+      error: applicationError,
+      stale: applicationStale,
+    },
+  ] = useGetApplicationQuery({
+    requestPolicy: "cache-first",
+    variables: {
+      id: applicationId || "",
+    },
+  });
+  const [{ data: userData, fetching: userFetching, error: userError }] =
+    useGetMeQuery();
+
+  const navigate = useNavigate();
+  const { followingPageUrl } = useApplicationContext();
+  const { applicantDashboard } = useFeatureFlags();
+  const [, executeMutation] = useUpdateUserAndApplicationMutation();
+  const cancelPath = applicantDashboard ? paths.dashboard() : paths.myProfile();
+  const nextStep = followingPageUrl ?? cancelPath;
+
+  const application = applicationData?.poolCandidate;
+  const resolvedIndigenousCommunities = unpackMaybes(
```

Comment:
unpackMaybes erases the distinction between `null` (a new user) and `[]` (a user who has previously claimed not to be indigenous). I think we should properly consider the null state, and leave the radio button undefined on first visit, instead of letting it default to "I am not a member of an indigenous group..." as it does right now.

Reply:
I'll ask Brooke on Slack.

Reply:
I've asked Brooke and she agrees with Tristan - leave the form unselected if the user has not yet made a selection.  Updated in [allow null to leave uninitialized](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6602/commits/0511f2c8f67aa31d035ee84990142b749360a1b2).